server/utils/response.js

```diff
@@ -48,12 +50,21 @@ export const setContentTypeFile = (res, fileName, fileSize = null, contentType =
   }
 }
 
-export const sendFile = (res, name, content, size) => {
+export const sendFileContent = (res, name, content, size) => {
   setContentTypeFile(res, name, size)
   res.write(content, 'binary')
   res.end(null, 'binary')
 }
 
+export const sendFile = ({ res, filePath, fileNameOutput = null }) => {
```

Comment:
cant they be { res, path, name = null } ?

Reply:
fileNameOutput is more explicit, because it's the name that you want the user to see when the file is dowloaded

Reply:
I would understand so even if you call the parameter fileName

Reply:
done